docs/source/index.rst

```diff
@@ -38,16 +41,21 @@ This is the documentation for Mantid |release|.
    :width: 200px
    :align: right
 
-* :ref:`Training`
+* :ref:`training`
+    - `Mantid Basic Course <http://www.mantidproject.org/Mantid_Basic_Course>`_
+    - `Introduction To Python <http://www.mantidproject.org/Introduction_To_Python>`_
+    - `Python in Mantid <http://www.mantidproject.org/Python_In_Mantid>`_
+    - `Extending Mantid With Python <http://www.mantidproject.org/Extending_Mantid_With_Python>`_
+    - :ref:`ill_training`
 * :ref:`Algorithms List`
 * :ref:`concepts contents`
 * :ref:`interfaces contents`
-* :ref:`Fit Functions List`
-* :ref:`fitminimizers contents`
+* :ref:`fitting contents`
+    - :ref:`Fit Functions List`
+    - :ref:`fitminimizers`
 * :ref:`techniques contents`
 * :ref:`api`
     - :ref:`Python <pythonapi>`
     - `C++ <http://doxygen.mantidproject.org/>`_ (Doxygen)
-* :ref:`Gallery <plotting>`
+* :ref:`Matplotlib Plot Gallery and Examples <plotting>`
```

Comment:
Maybe there should be a "Mantid" somewhere in this name, since all the examples include plotting workspaces

Reply:
I will do; it is not the matplotlib.org gallery after all!